Makefile

```diff
@@ -184,7 +184,11 @@ travis-test-snuba: test-snuba
 travis-test-js: test-js
 travis-test-cli: test-cli
 travis-test-dist:
-	SENTRY_BUILD=$(TRAVIS_COMMIT) SENTRY_LIGHT_BUILD=0 python setup.py sdist bdist_wheel
+	# NOTE: We quiet down output here to workaround an issue in travis that
```

Comment:
Do we know what the output is that is causing this issue?

Reply:
I spent a while on this last night. This command produces a LOT of output, honestly I'm not quite sure what has changed that causes the EAGAIN error but didn't want to spend more time trying to narrow it down. Presumably something somewhere is putting stdout into non-blocking mode.

Reply:
Can we compare stdout when running locally?

Reply:
Yeah I'll take a look